### ISSUE TITLE
AMD vs Apache License terms

### ISSUE THREAD

**Issue by alexsorkin** [Syncleus/aparapi#2]
Hi,

I would like to use this project as (de-facto) only continuation of initial AMD's @grfrost effort.
Is it possible to clear AMD vs Apache license compatibility issue?
In my understanding @grfrost is no more working for AMD, but moved to Google.
Would be great of AMD to relicense initial effort under Apache 2 terms.

Thanks.

**Comment by freemo** [comment]
Hi @alexsorkin ,

From what my lawyers tell me the license that AMD released this under is compatible with the Apache license version 2 already. As such permission from AMD does not need to be granted to re-license it under the Apache license version 2. The important part is just ensuring any attributions remain (copyright notices mostly, but also the names of contributors). But that is also a stipulation of the Apache license version 2. 

So long story short, this project already moved to Apache license version 2, and the old license is only kept for reference. It doesnt take any participation on the part of AMD to make that change.

**Comment by freemo** [comment]
@alexsorkin Is there anything else I can do for you, do you mind if I close this ticket?

Final word seems to be that the licenses are compatible so there is implied permission already given.

**Comment by freemo** [comment]
For future reference this file is specified and needed to satisfy the old license, otherwise the Apache License 2 is a compatible license and will be the license moving forward.

https://github.com/Syncleus/aparapi/blob/master/ATTRIBUTIONS.md

**Comment by freemo** [comment]
@alexsorkin  closing this until/if i hear back from you.

**Comment by alexsorkin** [comment]
Thank much, I've answered my question.

2016-12-03 21:45 GMT+02:00 Jeffrey Phillips Freeman <
notifications@github.com>:
